bevy_lint/tests/ui/main_return_without_appexit/muted_expr.rs

```diff
@@ -0,0 +1,19 @@
+//! A version of `main.rs` where the lint is muted on the expression. Since this lint is run on
+//! functions, it will still error.
+//!
+//! This behavior is misleading, so this test ensures it is not accidentally fixed without updating
```

Comment:
Maybe we should open a ticket for this and use the `bug_X` format as for the other ones

Reply:
I opened #132! Let me rename these to follow the same format.